docs/source/conf.py

```diff
@@ -131,12 +131,11 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = "_static/tephi-logo-200-137.png"
+html_logo = "_static/tephi-logo.svg"
 
 # The name of an image file (within the static path) to use as favicon of the
-# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
-# pixels large.
```

Comment:
This hasn't been true for years for most web browsers. The Iris Readthedocs page uses an SVG and that works just fine.